MBBSEmu/MEMORYMAP.MD

```diff
@@ -0,0 +1,32 @@
+﻿# MBBSEmu Memory Map
+Each Module in MBBSEmu is provided a dedicated, full 16-bit address space. Meaning each module is capable of addressing up to 16GB of Memory per module. 
```

Comment:
16-bit address space means 64k to me, so I'm not entirely sure how you get up to 16GB. Even with 64k segments to play with, that's still only 4GB. Maybe provide some more reasoning here, especially for the non-computer scientists who are interested in learning more

Reply:
Yeah, this was a mis-calculation. It is 4GB -- I'll update it.

Reply:
Yeah, my bad on that one. Calculated it wrong, 0xFFFF * 0xFFFF == 4GB, you're correct. I'll update the documentation there.

I'll give some context around how MBBSEmu handles memory as well.